api/routes/socket.js

```diff
@@ -23,7 +23,10 @@ module.exports = function (app, server) {
 				socketController.test(action.message);
 				break;
 			case 'socket/ticket/new':
-				socketController.addTicket(action.ticket);
+				socketController.addTicket(action.ticket).catch(err => {
+					console.error(err);
```

Comment:
No `console` please. I said add an empty `catch` callback

Reply:
I said it feels kinda weird to have an empty catch block. And does it matter since we have console.errors elsewhere anyways

Reply:
Empty catch block is actually an established practice for promises, and that's what the TODO comment is for anyway :)

I thought I got rid of most `console.error`s in the server. TBH `console.error`s are fine for debugging, but we should have all the errors handled in one place where `console.error` is used.